third_party/proto/cosmos-proto/cosmos.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package cosmos_proto;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/regen-network/cosmos-proto";
+
+extend google.protobuf.MessageOptions {
+  string interface_type = 93001;
```

Comment:
can we add a comment here or it's described in the README in github.com/regen-network/cosmos-proto ?

Reply:
removed this file as we will manually have to generate the switch statements